src/DependencyInjection/CmfSonataAdminIntegrationExtension.php

```diff
@@ -68,4 +89,22 @@ public function getXsdValidationBasePath()
     {
         return __DIR__.'/../Resources/config/schema';
     }
+
+    private function defineFactories(ContainerBuilder $container)
```

Comment:
or we can make the extension self-configuring, but then we should also create the factories if additional factories have been registered with the public method. or remove the public method as i would think that if people do additional things, they can just use the DI of their own bundle and don't need to interact with this bundle